src/components/playback/playbackmanager.js

```diff
@@ -309,13 +309,11 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
             if (codecProfile.Type === 'Audio') {
                 (codecProfile.Conditions || []).map(function (condition) {
                     if (condition.Condition === 'LessThanEqual' && condition.Property === 'AudioBitDepth') {
-                        maxAudioBitDepth = condition.Value;
-                    }
-                    if (condition.Condition === 'LessThanEqual' && condition.Property === 'AudioSampleRate') {
-                        maxAudioSampleRate = condition.Value;
-                    }
-                    if (condition.Condition === 'LessThanEqual' && condition.Property === 'AudioBitrate') {
-                        maxAudioBitrate = condition.Value;
+                        return maxAudioBitDepth = condition.Value;
+                    } else if (condition.Condition === 'LessThanEqual' && condition.Property === 'AudioSampleRate') {
+                        return maxAudioSampleRate = condition.Value;
+                    } else if (condition.Condition === 'LessThanEqual' && condition.Property === 'AudioBitrate') {
+                        return maxAudioBitrate = condition.Value;
```

Comment:
Is this equivalent in JavaScript?

Reply:
So the if...if...if, in this case, translates well to an if...else if...else if.

Ideally, in the future, we'll use [pattern matching](https://github.com/tc39/proposal-pattern-matching).

As for the return, the map() expects one. Returning the assignment will execute it (thus assigning the value) and fix the warnings on a map() function without any return.